examples/authentication/antd/src/App.tsx

```diff
@@ -39,20 +39,23 @@ const App: React.FC = () => {
 
             return Promise.reject();
         },
+        // eslint-disable-next-line
         register: (params: any) => {
             if (params.email && params.password) {
                 localStorage.setItem("email", params.email);
                 return Promise.resolve();
             }
             return Promise.reject();
         },
+        // eslint-disable-next-line
         updatePassword: (params: any) => {
             if (params.newPassword) {
                 //we can update password here
                 return Promise.resolve();
             }
             return Promise.reject();
         },
+        // eslint-disable-next-line
         forgotPassword: (params: any) => {
```

Comment:
```suggestion
        register: (params) => {
            if (params.email && params.password) {
                localStorage.setItem("email", params.email);
                return Promise.resolve();
            }
            return Promise.reject();
        },
        updatePassword: (params) => {
            if (params.newPassword) {
                //we can update password here
                return Promise.resolve();
            }
            return Promise.reject();
        },
        forgotPassword: (params) => {
```

Hey @sevkioruc, `AuthProvider` already supplies the types for these properties, even though they are all `any`s, we can just remove the `: any` notations to get rid of the warning, rather than adding `eslint-disable-next-line`.

This also applies to other examples under the `/authentication` directory. 

Can you update them as well? Sorry if I sound so nitpicky 😅

Reply:
Hi @aliemir , 
I will update them as soon as possible.
hahah don't worry. it is right :sweat_smile:

Reply:
done